Bio/PDB/Vector.py

```diff
@@ -10,194 +10,8 @@
 import numpy
 
 
-def m2rotaxis(m):
```

Comment:
Why are you removing this function (and the others after it) from ``Bio.Vector``? They are part of the module's public API and should stay here.